django_app/redbox_app/templates/macros/chat-macros.html

```diff
@@ -16,7 +16,11 @@ <h3 class="iai-chat-bubble__sources-heading govuk-heading-s govuk-!-margin-botto
             <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-0">
                 {% for source in sources %}
                 <li class="govuk-!-margin-bottom-0">
+                  {% if source.get_status_text() == "Complete" %}
                     <a class="iai-chat-bubbles__sources-link govuk-link" href="{{ source.url }}">{{ source.original_file_name }}</a>
+                  {% else %}
+                    <p class="iai-chat-bubbles__sources-link">{{ source.original_file_name }}</p>
```

Comment:
Please can you make this a `<span>` rather than a `<p>`?
That will keep the vertical spacing consistent when there are multiple links, and I don't think it needs the paragraph semantic meaning.